source/includes/fact-filter-queries.rst

```diff
@@ -0,0 +1,11 @@
+Allows you to filter and limit the amount of entries displayed in 
+the :guilabel:`Query Converter` pane.
+
+
+.. procedure::
+   :style: normal
+
+   .. step:: Select your query
+      Enter a text string to match against the object name and SQL 
+      syntax in the :guilabel:`Filter` text box or click  
```

Comment:
It's not clear what you mean by "SQL Syntax" here. The filtering only matches on the name of the query as shown in the query list, but for an ad-hoc (pasted) query the name defaults to the full query text unless renamed.